pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -112530,7 +112530,6 @@ self: {
        description = "A static website compiler library";
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
@vaibhavsagar Could you send a follow-up PR removing this `hydraPlatforms =` line as well?  If you don't remove this, then Hydra won't build and distribute `haskellPackages.hakyll`.

Reply:
Oh, wow. Sorry … It's been a while since I have merged something into stable …

Reply:
Don't worry about it!  I often miss this as well.

Reply:
Thanks, I opened https://github.com/NixOS/nixpkgs/pull/104375.